### DISCUSSION TITLE
Understanding AlbumArtist to Artist fall back

### DISCUSSION THREAD

**Discussion by john-** [jcorporation/myMPD#1055] [Q&A] [answered]
myMPD version: 10.3.1

Based on [documentation](https://jcorporation.github.io/myMPD/references/tags) if the AlbumArtist tag is not populated than Artist tag will be used.

However, after enabling MusicBrainz tags (Settings|Cloud|Show MusicBrainz links), songs without the AlbumArtist tag show a hyphen ("-") instead of Artist tag.  The link works correctly:  when clicked it directs to the MusicBrainz page for the artist.
![mympd_musicbrainz_links](https://github.com/jcorporation/myMPD/assets/2359125/e7d2cecd-aa10-4e19-950d-c567a5974828)

I would have expected it to display the Artist tag but not sure what the expected behavior is.

**Comment by jcorporation** [comment, marked as answer]
This is a bug. Should be fixed with commit 5b43fd1ed8de3114501bea67a643d2acb0207d15.
